yaksh/templates/yaksh/error_template.html

```diff
@@ -0,0 +1,93 @@
+{% block css%}
+    <link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/dashboard.css" type="text/css" />
+{% endblock %}
+
+{% load custom_filters %}
+
+{% if error_message %}
+<div class="col-md-12">
```

Comment:
Can make this class name as `col-md-10 col-md-offset-2 main` so as to avoid overlapping of side bar.